cilium-sysdump/__main__.py

```diff
@@ -133,9 +142,30 @@ def parse_comma_sep_list(arg_string):
         namespace.hubble_relay_ns = args.hubble_relay_ns
         pass
 
+    log.debug('Fetching nodes to determine cluster size...')
+    nodes = utils.get_nodes()
+    if not nodes:
+        log.error('No nodes found')
+    if len(nodes) > 20:
+        log.warning((
```

Comment:
Food for thoughts: should we warn the user with this message only if none of the 3 nodes/since/size-limit options is set?

Reply:
Ah yes, good catch. That would make much more sense!

Reply:
Fixed.